cmd/dmsg-discovery/internal/store/redis.go

```diff
@@ -60,6 +61,10 @@ func (r *redisStore) SetEntry(ctx context.Context, entry *disc.Entry, timeout ti
 		return disc.ErrUnexpected
 	}
 
+	if entry.Server != nil {
+		timeout = dmsg.DefaultUpdateInterval
```

Comment:
We should probably set this value a little but higher in case there is some latency, right?

Reply:
You're right. I set it to 5 minutes.